ports/servoshell/build.rs

```diff
@@ -76,11 +84,7 @@ fn main() -> Result<(), Box<dyn Error>> {
         generate_egl_bindings(out);
     }
 
-    if let Err(error) = EmitBuilder::builder()
-        .fail_on_error()
-        .git_sha(true /* short */)
-        .emit()
-    {
+    if let Err(error) = emit_git_sha() {
```

Comment:
Can we keep the emit_git_sha logic inlined
```
let git_options = Git2Builder::default().sha(true /* short */).build()?;
if let Err(error) = Emitter::default()
     .fail_on_error()
     .add_instructions(&git_options)
     .map(|emitter| emitter.emit())
```

so that we don't have to explicitly depend on `anyhow`? I'm worried about dependabot PRs that might fail due to duplicate crate errors.

Reply:
I converted the error to a string to remove the dependency on anyhow. 
I think inlining wouldn't really help, since we don't want to return an error from `main`, but instead want to fallback to setting `VERGEN_GIT_SHA=nogit`.

Reply:
Thanks! 

> we don't want to return an error from main, but instead want to fallback to setting VERGEN_GIT_SHA=nogit.

ah right, I overlooked that GitBuilder's build() is also fallible.